app/api/retrieveBlogs/route.js

```diff
@@ -1,11 +1,17 @@
-// Pulling mock mongodb
-import mockCollection from "@/data/mockBlogs";
+import { collection, getDocs } from "firebase/firestore";
+
+import { firestore as db } from "@/firebase/firebase";
 
 export async function GET(request) {
-  // TODO: get a connection to MongoDb and return from proper collection
-  const mockCursor = { toArray: () => Promise.resolve(mockCollection) };
-  const mockCollectionObject = { find: () => mockCursor };
-  const documents = await mockCollectionObject.find().toArray();
+  const querySnapshot = await getDocs(collection(db, "blogs"));
```

Comment:
maybe just adding a limit for now, and then eventually adding filters (maybe getting the blogs with the most posts / most recent posts) somewhere down the line